src/main/resources/db/migration/V8_31__add_stakeholder_parameter_tables.sql

```diff
@@ -0,0 +1,22 @@
+    create table stakeholder_parameter (
```

Comment:
`IF NOT EXISTS`maybe?

Reply:
Changed scripts for 8.31, 8.33 and 8.34 to be more defensive.